components/Navbar/Rightnav.js

```diff
@@ -73,17 +93,17 @@ const RightNav = (props) => {
         </Link>
```

Comment:
Lines 82, 87 and 92:
You could write a separate handler function to close your navbar and pass it to the onClick prop on each StyledLink.
```
 const handleCloseMenu = () => {
    setOpen(false); 
  };

<StyledLink onClick={handleCloseMenu}>
```